EVALUATION.md

```diff
@@ -0,0 +1,65 @@
+The evaluation process is done through the evaluate function that is located in the evaluation.py script (https://github.com/lovodkin93/ucca/blob/master/ucca/evaluation.py).
```

Comment:
I would surround function names, class names, filenames and command line samples with backticks (``) so they are stylized as code.

Reply:
Additionally, use a relative link instead of an absolute URL to your GitHub (github.com/lovodkin93).

Reply:
And make the URL an actual link by using `[link title](link target)` instead of just putting the URL as text.